pulsar-broker/src/test/java/org/apache/pulsar/client/impl/MessageChunkingSharedTest.java

```diff
@@ -211,6 +212,32 @@ private static String createChunkedMessage(int numChunks) {
         return Schema.STRING.decode(payload);
     }
 
+    @Test
+    public void testDuplicateForChunkMessage() throws Exception {
+        String topicName = "persistent://my-property/my-ns/testDuplicateForChunkMessage";
+        String producerName = "test-producer";
+        pulsarClient = PulsarClient.builder().serviceUrl(pulsar.getBrokerServiceUrl()).build();
+        // consumer
+        Consumer<byte[]> consumer = pulsarClient
+                .newConsumer()
+                .subscriptionName("test-sub")
+                .topic(topicName)
+                .subscribe();
+        // producer
+        Producer<String> partProducer = pulsarClient
+                .newProducer(Schema.AVRO(String.class))
+                .producerName(producerName)
+                .topic(topicName)
+                .enableChunking(true)
+                .enableBatching(false)
+                .create();
+        int messageSize = 6000; // payload size in KB
+        String message = "a".repeat(messageSize * 1000);
+        partProducer.newMessage().value(message).send();
```

Comment:
Aren't we supposed to test dedup messages? Is this a validate dedup + chunking test?

Reply:
Oh, thanks for your reminder. 
I write this test in the TransactionTest at the beginning which enables the dedup default. 
And move the test to MessageChunkingSharedTest before committing. But the MessageChunkingSharedTest does not enable dedup.